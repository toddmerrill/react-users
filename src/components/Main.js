import React from 'react';

const Main = React.createClass({
    // header and footer will go in this component
    render() {
        return (
            <div>
                {this.props.children}
            </div>
        )
    }
})

export default Main;
