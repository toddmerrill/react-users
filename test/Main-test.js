import React from 'react'; 
import Main from '../src/components/Main.js'
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('Main renders correctly', () => {
  shallowRenderer.render(<Main />);
  const tree = shallowRenderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
