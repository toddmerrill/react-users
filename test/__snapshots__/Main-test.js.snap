// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `<div />`;
